librz/bin/pdb/tpi.c

```diff
@@ -2938,6 +2263,38 @@ int parse_tpi_stream(void *parsed_pdb_stream, RZ_STREAM_FILE *stream) {
 	return 1;
 }
 
+/**
+ * \brief Get SType that matches tpi stream index
+ *
+ * \param index TPI Stream Index
+ */
+RZ_API SType *get_stype_by_index(ut32 index) {
```

Comment:
What I meant by calling convention here is to have name something like `rz_bin_pdb_stype_by_index()`